workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/ScreenViewFactory.kt

```diff
@@ -31,3 +37,87 @@ public interface ScreenViewFactory<in RenderingT : Screen> : ViewRegistry.Entry<
     container: ViewGroup? = null
   ): View
 }
+
+/**
```

Comment:
Outside of making `getViewFactoryForRendering` non-public, no real changes in this code, just moved it from another location.